docs/pjrt.md

```diff
@@ -0,0 +1,142 @@
+# Experimental PjRt Runtime Support
+
+The PyTorch/XLA team is currently migrating from the currently-supported XRT
+runtime to the [PjRt
+runtime](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/compiler/xla/pjrt)
+used by [JAX](https://github.com/google/jax). Although PjRt may work on TPU v2
+and v3, we plan on making PjRt the officially supported runtime for PyTorch/XLA
+on TPU v4 and future generations of TPU.
+
+PjRt is available as an _experimental preview_ in PyTorch/XLA r1.13. The
+PyTorch/XLA team will provide limited support on a best-effort basis during this
+preview. If you encounter a bug with PjRt, please file an issue on GitHub with
+the `runtime` tag.
+
+**This preview is mainly targeted at TPU v4**. In most cases, we expect that you can
+re-use your existing PyTorch/XLA code for TPU v4 with no changes. You may be able to
+adapt your v2 or v3 workload to PjRt with some caveats (see below).
+
+## Quickstart
+
+To start using PjRt with PyTorch/XLA, all you need to do is set the
+`PJRT_DEVICE` environment variable. If you're working on a TPU v2 or v3, keep
+reading to learn about the differences between TPU v2 and v3 and v4.
+
+### CPU
+
+On any machine with PyTorch/XLA installed, you can run our MNIST example on CPU
+like this:
+
+```
+PJRT_DEVICE=CPU python3 xla/test/test_train_mp_mnist.py --fake_data
+```
+
+### v4 TPU
+
+To create a new TPU with PyTorch/XLA r1.13 installed:
+
+```
+gcloud alpha compute tpus tpu-vm create $USER-pjrt --accelerator-type=v4-8 --version=tpu-vm-v4-pt-1.13 --zone=us-central2-b --project=$PROJECT
+```
+
+On a v4-8, you can run our ResNet50 example like this:
+
+```
+git clone --depth=1 --branch r1.13 https://github.com/pytorch/xla.git
+PJRT_DEVICE=TPU python3 xla/test/test_train_mp_imagenet.py --fake_data --batch_size=256 --num_epochs=1
+```
+
+By default, PjRt will use all TPU chips. To use only one TPU chip, configure
+`TPU_PROCESS_BOUNDS` and `TPU_VISIBLE_CHIPS`:
+
+```
+TPU_PROCESS_BOUNDS=1,1,1 TPU_VISIBLE_CHIPS=0 PJRT_DEVICE=TPU python3 xla/test/test_train_mp_imagenet.py --fake_data --batch_size=256 --num_epochs=1
```

Comment:
what if user changed https://github.com/pytorch/xla/blob/master/test/test_train_mp_imagenet.py#L272 `nprocs` to 1?

Reply:
`nprocs` is actually ignored in the PJRT case right now. There's a feature request on me internally to implement a special case for `nprocs=1`

Reply:
can we update the doc to reflect that? User might try to set it and will be confused. Ideally we should throw an error now if user use PJRT and set `nprocs` to 1 on TPU, but we don't need to cherry pick this change now.

Reply:
Done.